server/Interfaces/IWaterLevelSettings.cs

```diff
@@ -0,0 +1,11 @@
+using WAMServer.Models;
+
+
+namespace WAMServer.Interfaces
+{
+    public interface IWaterLevelSettings
```

Comment:
Ik weet dat ik dit heb gemaakt, maar wellicht moeten we deze renamen naar IWaterLevelSettingsService